formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
+
+Implementation
+^^^^^^^^^^^^^^
+
+The `ome.xsd` schema file defines new type enumerations: UnitsLength,
+UnitsTime, UnitsPressure, UnitsAngle, UnitsTemperature,
+UnitsElectricPotential, UnitsPower and UnitsFrequency. These types are used
+throughout the model. Each new unit attribute is in a pair with an existing
+attribute, e.g. PinholeSize and PinholeSizeUnit. The Xsd-Fu application then
```

Comment:
Not sure this sentence is at all understandable unless you're familiar with the implementation though.  "processes..into java classes that use types based on ..."  It's not at all clear what this *really* means.  There are at least three things here: the unit classes, types based on the classes and the OME model objects which contain those types?  This could be split up into several sentences describing what each part is and how they are related.